fabric-gamerules-v1/src/main/java/net/fabricmc/fabric/mixin/gamerule/MixinBooleanRule.java

```diff
@@ -0,0 +1,15 @@
+package net.fabricmc.fabric.mixin.gamerule;
+
+import net.minecraft.world.GameRules;
+import org.apache.commons.lang3.NotImplementedException;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(GameRules.BooleanRule.class)
+public interface MixinBooleanRule {
+	@SuppressWarnings("PublicStaticMixinMember")
+	@Invoker
+	static GameRules.RuleType<GameRules.BooleanRule> callOf(boolean defaultValue) {
+		throw new NotImplementedException("Mixin to GameRules$BooleanRule Failed!");
```

Comment:
Just a matter of style, but empty mixin methods should generally throw `AssertionError`, as them being called simply cannot happen without a huge overlook. `NotImplementedException` implies the body of the method will be filled in later.

Reply:
Sounds good to me.